cellprofiler/modules/classifyobjects.py

```diff
@@ -247,110 +254,113 @@ def add_single_measurement(self, can_delete=True):
+*(Used only if "%(BC_EVEN)s" selected)*
+
+This is the threshold that separates the lowest bin from the others. The
+lower threshold, upper threshold, and number of bins define the
+thresholds of bins between the lowest and highest.
+""" % globals()))
 
         group.append("wants_low_bin", cps.Binary(
-                "Use a bin for objects below the threshold?", False, doc="""
-            Select <i>%(YES)s</i> if you want to create a bin for objects
-            whose values fall below the low threshold. Select <i>%(NO)s</i>
-            if you do not want a bin for these objects.
-            """ % globals()))
+                "Use a bin for objects below the threshold?", False, doc="""\
+Select "*%(YES)s*" if you want to create a bin for objects whose values
+fall below the low threshold. Select "*%(NO)s*" if you do not want a bin
+for these objects.
+""" % globals()))
 
         def min_upper_threshold():
             return group.low_threshold.value + np.finfo(float).eps
 
         group.append("high_threshold", cps.Float(
                 "Upper threshold", 1,
-                minval=cps.NumberConnector(min_upper_threshold), doc="""
-            <i>(Used only if "%(BC_EVEN)s" selected)</i><br>
-            This is the threshold that separates the last bin from
-            the others.
-            Note that if you would like two bins, you should select <i>%(BC_CUSTOM)s</i>.
-            """ % globals()))
+                minval=cps.NumberConnector(min_upper_threshold), doc="""\
+*(Used only if "%(BC_EVEN)s" selected)*
+
+This is the threshold that separates the last bin from the others. Note
+that if you would like two bins, you should select "*%(BC_CUSTOM)s*".
+""" % globals()))
 
         group.append("wants_high_bin", cps.Binary(
-                "Use a bin for objects above the threshold?", False, doc="""
-            Select <i>%(YES)s</i> if you want to create a bin for objects
-            whose values are above the high threshold. <br>
-            Select <i>%(NO)s</i> if you do not want a bin for these objects.""" % globals()))
+                "Use a bin for objects above the threshold?", False, doc="""\
+Select "*%(YES)s*" if you want to create a bin for objects whose values
+are above the high threshold.
+
+Select "*%(NO)s*" if you do not want a bin for these objects.
+""" % globals()))
 
         group.append("custom_thresholds", cps.Text(
                 "Enter the custom thresholds separating the values between bins",
-                "0,1", doc="""
-            <i>(Used only if "%(BC_CUSTOM)s" selected)</i><br>
-            This setting establishes the threshold values for the
-            bins. You should enter one threshold between each bin, separating
-            thresholds with commas (for example, <i>0.3, 1.5, 2.1</i> for four bins).
-            The module will create one more bin than there are thresholds.
-            """ % globals()))
+                "0,1", doc="""\
+*(Used only if "%(BC_CUSTOM)s" selected)*
```

Comment:
Please change to  `*(Used only if using a single measurement and "%(BC_CUSTOM)s" selected)*`